### ISSUE TITLE
When both external and duplicate are added, beep boop removes both labels and resets to open state.

### ISSUE THREAD

**Issue by ChristianMurphy** [unifiedjs/beep-boop-beta#21]
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 

### Steps to reproduce

see: https://github.com/remarkjs/react-markdown/issues/641

### Expected behavior

One or both labels should be kept, and phase should move to phase/no

**Comment by wooorm** [comment]
yeah the thinking is that only one reason exists per a category: semver/major *or* semver/minor. open/multiple *or* open/needs-info, no/wontfix or no/external
